mapie/tests/test_conformity_scores.py

```diff
@@ -382,18 +382,17 @@ def test_residual_normalised_prefit_get_estimation_distribution() -> None:
 @pytest.mark.parametrize("score", [AbsoluteConformityScore(),
                                    GammaConformityScore(),
                                    ResidualNormalisedScore()])
-@pytest.mark.parametrize("alpha", [[0.3], [0.5, 0.4]])
+@pytest.mark.parametrize("alpha", [[0.5], [0.5, 0.6]])
 def test_intervals_shape_with_every_score(
     score: ConformityScore,
     alpha: Any
```

Comment:
Why is this `Any`?

Reply:
Good question!

Reply:
I have changed it to `NDArray`

Reply:
Can't it be a `float` or `NDArray`?

Reply:
On line 385, we only have an array of floats. This is just in the context of the test, so it's not a problem for MAPIE's main content.